.github/workflows/ci.yml

```diff
@@ -19,6 +19,11 @@ jobs:
     - name: setup nvm
       shell: bash -l -eo pipefail {0}
       run: nvm install 8.10.0 && nvm use 8.10.0
+    - name: setup npm
+      shell: bash -l -eo pipefail {0}
+      run: |
+        npm config set spin false
+        npm install -g npm
```

Comment:
Please move these steps in the shell scripts that does the checks so that we don't have to store `package-lock.json` in the repo.